index.js

```diff
@@ -0,0 +1,6 @@
+'use strict';
+
+const greet = require('./lib/greet.js');
+
```

Comment:
looks good.

for the bonus you could have included `let name = process.argv[2];` and then pass the variable name into `greet.holla()` instead of passing in `'darcy'`.